config/rbac/hive_reader_role.yaml

```diff
@@ -68,3 +68,13 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - "configmaps"
+  resourceNames:
+  - "install-log-regexes"
+  verbs:
+  - get
+  - patch
+  - update
```

Comment:
Can this limit just to the hive namespace?

Reply:
I've done this with a separate rolebinding but it may not be necessary with our plans to integrate with config management.

Reply:
Still a role change in here. :)